.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,63 @@
 ## PR Summary
 
-## PR Checklist
-
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way
-
 <!--
-Thank you so much for your PR!  To help us review your contribution, please
-consider the following points:
+Thank you so much for your PR! Please summarize the purpose of the PR here
+using at least 1-2 sentences.
```

Comment:
Going back and forth on "Please summarize the purpose of the PR here using at least 1-2 sentences." being necessary since this is now kinda in the list but also 🤷‍♀️

Reply:
Oh I actually removed it from the list below because now that the list is up here, it looked *really* redundant. Hopefully that helps!